Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.ShowCase.ts2

```diff
@@ -0,0 +1,22 @@
+/**
+* NodeTypes around Show Cases
+*/
+
+
+prototype(Neos.NeosIo:Reference.ShowCase.Document) < prototype(Neos.NeosIo:Reference.Document) {
+	body {
+		content {
+			main = Neos.NeosIo:Reference.ShowCase
```

Comment:
Oh there is a bug with the project type name, I'll fix that.
But as written in my commit message it is on purpose that the page is currently not styled and only shows the basic information.
The later goal would be to use it as detail page and in case of a case study that detail page would be the case study. So I would not waste too much time now into styling a page which for the moment won't be shown, but is is prepared for later.
The documents however should be added as hidden pages.